howto/manage/setup-tls-for-appliance.md

```diff
@@ -0,0 +1,85 @@
+
+## Add a DNS Record
+
+Setting up DNS redirection depends on your DNS provider. Refer to the documentation of your provider to create a DNS record pointing to the IP/DNS of the AWS instance where the Anbox Cloud appliance is running on.
+
+## Configure the location
+
+Configure the location for the appliance using the created DNS name.
+
+    sudo snap set anbox-cloud-appliance experimental.location=<your DNS name>
+
+[note type="information" status="Note"]This option is experimental and will be removed in the future release when a better replacement exists.[/note]
+
+## Generate an SSL certificate
+
+There are many ways to create a valid SSL certificate. One way is to use [Let's Encrypt](https://letsencrypt.org/) to generate a free SSL certificate.
+
+First, connect and SSH into your appliance instance, and install the `certbot` snap:
+
+    sudo snap install --classic certbot
+
+Before generating your certificate using `certbot`, you need to stop the traefik service from listening on port 80 for the certificate creation.
+
+    sudo snap stop anbox-cloud-appliance.traefik
+
+Then run the following command to generate your certificate:
+
+    sudo certbot certonly --standalone
+
+It prompts you to enter the domain name for the certificate to be generated. You will see the following message when the certificate is created successfully.
+
+```bash
+Successfully received certificate.
+Certificate is saved at: /etc/letsencrypt/live/<your domain name>/fullchain.pem
+Key is saved at:         /etc/letsencrypt/live/<your domain name>/privkey.pem
+This certificate expires on yyyy-MM-dd.
+These files will be updated when the certificate renews.
+Certbot has set up a scheduled task to automatically renew this certificate in the background.
+```
+
+## Install an SSL certificate
+
+Copy the generated certificate to the `/var/snap/anbox-cloud-appliance/common/traefik/tls` directory
+
+    sudo cp /etc/letsencrypt/live/<your domain name>/fullchain.pem /var/snap/anbox-cloud-appliance/common/traefik/tls/cert.pem
+    sudo cp /etc/letsencrypt/live/<your domain name>/privkey.pem   /var/snap/anbox-cloud-appliance/common/traefik/tls/key.pem
+
+Start the traefik service afterwards
+
+    sudo snap start anbox-cloud-appliance.traefik
+
+With the certificate installed on the appliance, you now can access it using the created domain name.
+
+## Renew an SSL certificate
+
+The `certbot` snap packages installed on your machine already setup a systemd timer that will renew your certificate automatically before they expire. However in order to get the certificate renewed successfully for the appliance, you need to
+
+1. Stop traefik service to release port 80 right before the certificate is going to be renewed. This can be done through the `pre-hook`:
```

Comment:
```suggestion
1. Stop the traefik service to release port 80 right before the certificate is going to be renewed. This can be done through the `pre-hook`:
```